README.md

```diff
@@ -166,16 +166,17 @@ then Lodash/Underscore is the better option.*
 
 **[String](#string)**
 
+1. [_.endsWith](#endsWith)
 1. [_.padStart and _.padEnd](#_padstart-and-_padend)
 1. [_.repeat](#_repeat)
 1. [_.replace](#_replace)
 1. [_.split](#_split)
 1. [_.startsWith](#startsWith)
-1. [_.template](#_template)
-1. [_.toLower](#_tolower)
-1. [_.toUpper](#_toupper)
+2. [_.template](#_template)
```

Comment:
Why changing these numbers?

Reply:
@stevemao fixed it, must have auto incremented from my IDE when I was moving endsWith to be alphabetically in order